ocs_ci/utility/rosa.py

```diff
@@ -133,11 +151,12 @@ def appliance_mode_cluster(cluster_name, ocp_version, region):
     addon_name = config.ENV_DATA.get("addon_name", "")
     size = config.ENV_DATA["size"]
     public_key = config.AUTH.get("managed_service", {}).get("public_key", "")
-    subnet_ids = config.ENV_DATA["subnet_ids"]
     notification_email_0 = config.REPORTING.get("notification_email_0")
     notification_email_1 = config.REPORTING.get("notification_email_1")
     notification_email_2 = config.REPORTING.get("notification_email_2")
     region = config.ENV_DATA.get("region", "")
+    private_link = config.ENV_DATA.get("private_link", "")
```

Comment:
Same as above, `ENV_DATA["private_link"]` is used as boolean, so I would suggest to not set string as default value:

```suggestion
    private_link = config.ENV_DATA.get("private_link")
```

Reply:
Updated as below
`private_link = config.ENV_DATA.get("private_link", False)`